test/integration/api.js

```diff
@@ -0,0 +1,319 @@
+const axios = require('axios');
+const fs = require('fs');
+const FormData = require('form-data');
+
+class API {
+    constructor(config) {
+        this.config = config
+      }
+      
+    async createEntity(appUrl, serviceName, entityName, srvpath){
+        let localConfig;
+        let response;
+        let incidentID;
+        localConfig = JSON.parse(JSON.stringify(this.config));
```

Comment:
This is used in all the methods. Can you set it in constructor and use it wherever required?